k-distribution/tutorial/find-k.mak

```diff
@@ -1,10 +1,12 @@
 MAKEFILE_PATH := $(dir $(abspath $(lastword $(MAKEFILE_LIST))))
-ifneq (,$(wildcard ${MAKEFILE_PATH}/../include/ktest.mak))
+ifneq (,$(wildcard ${MAKEFILE_PATH}/../../../include/kframework/ktest.mak))
+export K_HOME?=${MAKEFILE_PATH}/../../../
+else ifneq (,$(wildcard ${MAKEFILE_PATH}/../include/kframework/ktest.mak))
 export K_HOME?=${MAKEFILE_PATH}/..
-else ifneq (,$(wildcard /usr/lib/kframework/include/ktest.mak))
-export K_HOME?=/usr/lib/kframework
-else ifneq (,$(wildcard /usr/local/lib/kframework/include/ktest.mak))
-export K_HOME?=/usr/local/lib/kframework
+else ifneq (,$(wildcard /usr/include/kframework/ktest.mak))
+export K_HOME?=/usr
+else ifneq (,$(wildcard /usr/local/include/kframework/ktest.mak))
+export K_HOME?=/usr/local
```

Comment:
Why do we need to check so many places? I thought that this was to make the installation point standard across all distributions? Is the `/usr/include/kframework` vs `/usr/local/include/kframework` a difference between Linux/MacOS.

I guess the point is more to make it so that we're not doing everything under `/usr/lib/kframework/...`, but instead directly inside `/usr/...` (on Linux) or `/usr/local/...` (on MacOS)?

Reply:
These are the cases for if the tutorial is run from its github sources, its maven target directory, or from a debian package, or from mac os.